generate.sh

```diff
@@ -6,41 +6,44 @@ rm -rf client
 # This should be another step
 #curl -s http://localhost:4242/docs/openapi.json | jq > openapi.json
 # get enterprise version
-curl -s https://app.unleash-hosted.com/hosted/docs/openapi.json | jq > openapi.json
+# curl -s https://us.app.unleash-hosted.com/ushosted/docs/openapi.json | jq > openapi.json
 
-# Delete some schemas and paths using those schemas
-cat openapi.json | \
-jq 'del(.components.schemas | (
-    .changeRequestsSchema,
-    .changeRequestSchema,
-    .changeRequestFeatureSchema,
-    .changeRequestEventSchema,
-    .changeRequestOneOrManyCreateSchema
-))' | \
-jq 'del(.paths | (
-."/api/admin/projects/{projectId}/change-requests",
-."/api/admin/projects/{projectId}/change-requests/config",
-."/api/admin/projects/{projectId}/change-requests/open",
-."/api/admin/projects/{projectId}/change-requests/pending",
-."/api/admin/projects/{projectId}/change-requests/pending/{featureName}",
-."/api/admin/projects/{projectId}/change-requests/{changeRequestId}/changes/{changeId}",
-."/api/admin/projects/{projectId}/change-requests/{id}",
-."/api/admin/projects/{projectId}/change-requests/{id}/comments",
-."/api/admin/projects/{projectId}/change-requests/{id}/state",
-."/api/admin/projects/{projectId}/change-requests/{id}/title",
-."/api/admin/projects/{projectId}/environments/{environment}/change-requests",
-."/api/admin/projects/{projectId}/environments/{environment}/change-requests/config"
-))' > modified-openapi.json
+# # Delete some schemas and paths using those schemas
+# cat openapi.json | \
+# jq 'del(.components.schemas | (
+#     .changeRequestsSchema,
+#     .changeRequestSchema,
+#     .changeRequestChangeSchema,
+#     .changeRequestSegmentChangeSchema,
+#     .changeRequestCreateSegmentSchema,
+#     .changeRequestCreateSchema,
```

Comment:
More change request schemas to ignore. We should isolate the problem and propose a solution